src/main/java/atmosfears/AtmosFearsBE/model/Location.java

```diff
@@ -0,0 +1,26 @@
+package atmosfears.AtmosFearsBE.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "locations")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Location {
+    @Id
+    private String id;
+
+    private String code;
+    private Double N;
```

Comment:
Tbh, I don't like this short field naming. What about changing it to (for example) `northCoordinate` and `eastCoordinate`

Reply:
Also what is `code`?